### ISSUE TITLE
[Build] Current localization method overwrite PhoneGap default localization template

### ISSUE THREAD

**Issue by amirudin** [phonegap/build#199]
@hardeep introduce the following localization method on Build:
Ref: http://community.phonegap.com/nitobi/topics/problem_with_binary_details_ios_localization_builded_with_phonegap_build#reply_6687343

`- locales
--- en
---- local.strings
--- fr
---- local.strings`

However, it seems to overwrite default localization available set by PhoneGap itself:
The following default template default template the become empty (blank).

https://github.com/phonegap/phonegap/tree/master/lib/ios/bin/templates/project/__TESTING__/Resources

**Comment by networksolution** [comment]
Hi. It is not possible to have the photoshooting UI (Camera plugin) in iPhone in Italian like the rest of the phone and the app?

**Comment by wildabeast** [comment]
@networksolution yes it is possible, i've outlined how [here](http://community.phonegap.com/nitobi/topics/adding-supported-languages-to-your-iphone-app-on-build)

**Comment by networksolution** [comment]
Sorry but at the link you provided, the forum stripped away the tags from your reply...
Where to insert the string "it" in config.xml? It is not specified.

**Comment by wildabeast** [comment]
Thanks for pointing that out -- here's how it should read:

    <gap:config-file platform="ios" parent="CFBundleLocalizations" mode="replace"> 
        <array>
          <string>it</string>
          <string>fr</string>
        </array>
    </gap:config-file>

Throw that in your PGB config.xml and when you change your iPhone to italian or french, the camera plugin should render in that language.

**Comment by artlogic** [comment]
@wildabeast Forgive me because it's been nearly 18 months, but does this fix solve the root issue here, or is that still a problem?

**Comment by wildabeast** [comment]
@networksolution's question was actually a different issue.

@artlogic I can look into the main issue here, if it would still be of any value to you..

**Comment by artlogic** [comment]
@wildabeast If you have a few spare moments it would be nice to get it fixed, but frankly, it was @networksolution's question that even got me thinking about it again. I haven't had a user complaining about it for awhile now.

**Comment by wildabeast** [comment]
cool, i'll try to have a look.

**Comment by artlogic** [comment]
Much appreciated! :+1:

**Comment by bmoshergit** [comment]
Any chance we can get similar support for Android for the Google Play store?  Right now there's no way for PhoneGap Build apps to tell Google Play that they support more than one language.

Here's a related ticket: https://github.com/phonegap/build/issues/401

**Comment by networksolution** [comment]
This should not be an issue for Android. Only the Apple Store displays languages. Google Play won't display any language list when downloading an app, but this is native behaviour, and it is correct

**Comment by bmoshergit** [comment]
@networksolution I'm not sure I know what you mean.  When you list your app in Google Play, Google auto-detects what languages you support in the app, by looking for the localized files.  Unfortunately Google thinks my app built with PhoneGap Build only supports English even though it actually support 4 other languages.  Google keeps sending me "tips" saying that they see I've provided descriptions and images in other languages my Google Play listing and that I can increase my downloads by translating the app to those other languages too.

**Comment by azabala** [comment]
@networksolution , @bmoshergit  is right. It seems Google Play search in /res folder for at least a value* folder for each supported language. For example, if your default language is english, it looks for a /res/value folder. If you add a spanish language, it looks for a /res/values-es folder

**Comment by regnete** [comment]
+1 for an Android solution

**Comment by Reon90** [comment]
+1
